Update README.md with Exposed Port check as part of "getting started" (#338)
signalfx/splunk-otel-collector: README.md

- [Splunk Access Token](https://docs.splunk.com/Observability/admin/authentication-tokens/org-tokens.html#admin-org-tokens)
- [Splunk Realm](https://dev.splunk.com/observability/docs/realms_in_endpoints/)
- [Agent or Gateway mode](docs/agent-vs-gateway.md)

> - [Doublecheck exposed ports](https://github.com/signalfx/splunk-otel-collector/blob/main/docs/security.md#exposed-endpoints) to make sure your environment doesn't have conflicts- these can be changed via Collector Agent's configuration

Reviewer: ```suggestion
- [Double-check exposed ports](https://github.com/signalfx/splunk-otel-collector/blob/main/docs/security.md#exposed-endpoints) to make sure your environment doesn't have conflicts. Ports can be changed in the collector's configuration.
```

Author: thumbsup
